src/receiving_queue/ReceivingQueue.jsx

```diff
@@ -160,6 +213,10 @@ const ReceivingQueue = () => {
     [filteredReceivingQueue, selectedShipmentIds]
   );
 
+  useEffect(() => {
+    if (isMounted) reloadQueueData();
```

Comment:
Do we only want to load if we mount the queue table itself? Also we may want to clean up here as we did before to prevent this state from getting weird. `return () => setIsMounted(false);`